examples/vectoradd_jax/tesseract_api.py

```diff
@@ -14,7 +15,7 @@
 
 
 class Vector_and_Scalar(BaseModel):
-    v: Differentiable[Array[(None,), Float32]] = Field(
+    v: Annotated[np.ndarray, Differentiable[Array[(None,), Float32]]] = Field(
```

Comment:
New behavior is tested here. TBD how much we should expand this to other examples + docs. I'd be cool with using this consistently throughout; other opinions welcome.